src/_variables.scss

```diff
@@ -271,4 +278,3 @@ $carousel-s2-tablist-thumbnail-active-border-width: 10px !default;
 $txthl-bg-color: #ff0 !default;
 $txthl-color: #000 !default;
 $txthl-font-weight: 700 !default;
-
```

Comment:
Are you using the EditorConfig plug-in for Atom? Can't remember if it actually worked, but I'm sure there was one.